Screens/filter.css

```diff
@@ -0,0 +1,84 @@
+.comp-header .filter-form{
+    position: absolute;
+    top:100%; right:-110%;
+    width:40rem;
+    box-shadow: var(--box-shadow);
+    padding:1.5rem;
+    border-radius: .5rem;
+    background: var(--white);
+    text-align: center;
+}
+
+.comp-header .filter-form.active{
+    right:1rem;
+    transition: .4s linear;
+}
+
+.filter-form .filter-title{
+    display: flex;
+    font-size: 1.8rem;
+    font-weight: bold;
+    justify-content: space-between;
+}
+
+.filter-form .filter-category{
+    font-size: 1.4rem;
+    text-align: start;
+    padding-left: 1rem ;
+}
+
+.clear-filter {
+    color: black;
```

Comment:
Color variables would be preferred here.

Reply:
yes i will do